node-js-basics-101/node_modules/node-static/test/integration/node-static-test.js

```diff
@@ -0,0 +1,390 @@
+var vows    = require('vows')
+  , request = require('request')
```

Comment:
__[jshint]__
* Comma warnings can be turned off with 'laxcomma'.
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->